sdk/core/azure_core/Cargo.toml

```diff
@@ -50,6 +50,7 @@ reqwest_gzip = ["typespec_client_core/reqwest_gzip"]
 reqwest_rustls = ["typespec_client_core/reqwest_rustls"]
 test = []
 tokio_fs = ["typespec_client_core/tokio_fs"]
+tokio_process = ["dep:tokio", "tokio/process"]
 tokio_sleep = ["typespec_client_core/tokio_sleep"]
```

Comment:
Note to self: we're creating enough `tokio`-based features that maybe we should have a `tokio` feature that pulls them all in.